app/services/search/view-search-results.service.js

```diff
@@ -39,15 +30,15 @@ const RETURN_REFERENCE_PATTERN = /^[1-9]\d*$/
 async function go(searchQuery, page) {
   const pageNumber = Number(page)
 
-  const licenceSearchResult = await _searchLicences(searchQuery, pageNumber)
+  const licenceSearchResult = await FetchLicenceSearchResultsService.go(searchQuery, pageNumber)
```

Comment:
I would suggest doing the following


```suggestion
  const { results: licences, total: licencesTotal } = await FetchLicenceSearchResultsService.go(searchQuery, pageNumber)
```

Then there should be no need for the next line, and you can just reference `licencesTotal` in `Math.max(licenceSearchResult.total, returnLogSearchResult.total)` below.

Reply:
Done, including removing the subsequent line - I've pushed the check for empty results into the presenter